### ISSUE TITLE
BUG - DID Balance

### ISSUE THREAD

**Issue by rgazetta** [iNextrix/ASTPP#474]
Hi,

I have configured DID successfully in ASTPP 3.6
But issue is with out any balance in customer getting incoming calls. There is no blockage for incoming  in 0 balance account. Also included seconds is over but getting calls randomly.
Can you please tell me where is the issue , where i have miss configured.

Forum
https://groups.google.com/forum/#!topic/astpp/QxNtocm7jm0

**Comment by rgazetta** [comment]
Hello,

This BUG procures another failure, the connections are terminated without any reason, but if you remove the minute value registered in the number, the call works normally without any drops, would it be possible to verify this urgently?

**Comment by rgazetta** [comment]
This problem https://groups.google.com/forum/#!topic/astpp/HmMvFVPYQiw is this BUG

**Comment by Hemdip** [comment]
@rgazetta 

We would suggest you to upgrade with v4.0 as it more stable version and fixed so many issues. 

Regarding your concern in v3.6 there is one variable INB_Free that you have to true/false so it will goes to check balance for DID

**Comment by rgazetta** [comment]
Thanks for the response, but for the moment the upgrade is not an option.

The variable INB_Free, should be configures where and how?

**Comment by Hemdip** [comment]
You can do it from astpp.lua

Also i would like to share that similar kind of [issue](https://github.com/iNextrix/ASTPP/issues/476) already open

**Comment by rgazetta** [comment]
/var/lib/astpp/astpp.lua
I set INB_FREE="false"
Nor work, call continue in balance 0

**Comment by Hemdip** [comment]
@rgazetta Yes it will go to check Origination rates and apply accordingly. In your condition please check you have origination rate with 1.00/per minute.
